pkgs/stdenv/generic/default.nix

```diff
@@ -117,7 +117,8 @@ let
       # Utility flags to test the type of platform.
       inherit (hostPlatform)
         isDarwin isLinux isSunOS isHurd isCygwin isFreeBSD isOpenBSD
-        isi686 isx86_64 is64bit isArm isAarch64 isMips isBigEndian;
+        isi686 isx86_64 is64bit isArm isAarch64 isMips isBigEndian
+        isMusl isGlibc;
```

Comment:
I want to deprecate these in `18.03` to force people to think about cross. What do you think? If so, maybe we shouldn't add anymore here.

Reply:
Works for me! I'll make a pass removing these.  I do like having "isMusl" and "isGlibc"-- they're less error-prone than `libc == "stringvalue"`.  But accessing them from the appropriate platform (`hostPlatform`) sounds good to me.